src/tests/EventPipeTracee/Program.cs

```diff
@@ -16,10 +16,11 @@ class Program
 
         static void Main(string[] args)
         {
-            TestBody(args[0]);
+            bool spinWait10 = args.Length > 1 && args[1] == "SpinWait";
```

Comment:
Should we be using the .Equals method for string comparison?

Reply:
In this case, it is not necessary. If both the left and right side are `string`, the the `==` operator and the `Equals` method do the same thing. If, for example, the left side was of type `object` (even if the dynamic type is `string`), then using the `Equals` method would be required; the `==` operator would then be checking for reference equality.

But I can change this to be more consistent and unambiguous to read.

Reply:
While this will do a `StringComparison.Ordinal` comparison, shouldn't we still use `String.Equals` since it's recommended by .NET Best Practices? [https://docs.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings](https://docs.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings)